assets/css/style.css

```
@@ -37,22 +45,49 @@ h1 {
   color: rgb(163, 162, 162);
 
   font-size: 18px;
-  font-family: 'Fira Code', monospace;
+}
+
+.container_about_me {
+  margin-bottom: 30px;
+  margin-top: 30px;
+
+  font-weight: 600;
 }
 
 .container_title {
+  margin: 30px 0 10px 0;
   font-weight: 600;
+
+  text-align: left;
 }
 
-.container_about_me {
-  margin-top: 20px;
+.container_time {
+  margin-bottom: 10px;
+
+  color: rgb(163, 162, 162);
+}
+
+.container_block {
+  font-weight: 500;
+}
+
+.container_text {
+  margin-bottom: 10px;
+}
+
+.container_indent {
+ margin: 0 20px 0 20px;
```

```suggestion
  margin: 0 20px 0 20px;
```
be consistent

btw it can be done with only 
```margin: 0 20px;```